executor/runtime/docker/docker.go

```diff
@@ -1881,13 +1886,12 @@ func setupNetworkingArgs(burst bool, c *runtimeTypes.Container) []string {
 	if burst || c.TitusInfo.GetAllowNetworkBursting() {
 		args = append(args, "--burst=true")
 	}
-	if jumbo, ok := c.TitusInfo.GetPassthroughAttributes()[jumboFrameParam]; ok {
-		if val, err := strconv.ParseBool(jumbo); err != nil {
-			log.Error("Could not parse value for "+jumboFrameParam+": ", err)
-		} else if val {
-			args = append(args, "--jumbo=true")
-		}
+	if jumbo, err := c.Jumbo(); err != nil {
+		log.WithError(err).Error("Could not parse value for jumbo frame parameter")
+	} else if jumbo {
+		args = append(args, "--jumbo=true")
```

Comment:
So we have to have both things in place here:
* task attribute to allow jumbo
* New version of vpc service that sends routes with mtus

Anything else?

I'm trying to wrap my head around deployment strategy and potential failure modes between the components.

Reply:
The task attribute to allow jumbo has been a thing for a long time, but yeah, the thing that goes out "first" now is the VPC service that sends down routes. Earlier this was written with the idea that the agent would go out first before the VPC Service was taught to send down any routes.